packages/apps/kadena-docs/src/components/Analytics/Analytics.tsx

```diff
@@ -0,0 +1,61 @@
+import { analyticsPageView } from '@/utils/analytics';
+import { useRouter } from 'next/router';
+import Script from 'next/script';
+import React, { FC, useEffect } from 'react';
+
+const TRACKING_ID: string = process.env.NEXT_PUBLIC_TRACKING_ID ?? '';
+
+export const Analytics: FC = () => {
+  const router = useRouter();
+  // 👇 send page views when users gets to the landing page
+  useEffect(() => {
+    if (!TRACKING_ID) return;
+
+    gtag('config', TRACKING_ID, {
+      // eslint-disable-next-line @typescript-eslint/naming-convention
+      send_page_view: false, //manually send page views to have full control
+    });
+    gtag('event', 'page_view', {
+      page_path: window.location.pathname,
+      send_to: TRACKING_ID,
+    });
+  }, []);
```

Comment:
Do we look for DoNotTrack property/header/whatever? Or is this something we don't care about?